code_samples/api/public_php_api/src/Command/FormSubmissionCommand.php

```diff
@@ -40,9 +40,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $formValue = $content->getFieldValue('form', 'eng-GB')->getFormValue();
```

Comment:
`foreach ($formValue->getFields() as $field) {dump($field->getId(), $field->getIdentifier(), $field->getName());}` showed me what to use while testing this command for real.